src/main/java/org/czekalski/petapiintegration/resourceserver/model/Animal.java

```diff
@@ -0,0 +1,76 @@
+package org.czekalski.petapiintegration.resourceserver.model;
+
+public class Animal {
+    private Integer id;
+    private String name;
+    private String breed;
+    private String age;
+    private String gender;
+
+
+    private Animal(AnimalBuilder animalBuilder) {
+        this.id = animalBuilder.id;
+        this.name = animalBuilder.name;
+        this.breed = animalBuilder.breed;
+        this.age = animalBuilder.age;
+        this.gender = animalBuilder.gender;
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getBreed() {
+        return breed;
+    }
+
+    public String getAge() {
+        return age;
+    }
+
+    public String getGender() {
+        return gender;
+    }
+
+    public static class AnimalBuilder {
+        private Integer id;
+        private String name;
+        private String breed;
+        private String age;
+        private String gender;
+
+        public AnimalBuilder setId(Integer id) {
```

Comment:
```suggestion
        public AnimalBuilder withId(Integer id) {
```

Reply:
To samo dotyczy pozostałych metod buildera. Dlaczego tak? A to dlatego, że `setXXX` to jest pattern używany w POJO. Buildery zazwyczaj mają `withXXX` albo po prostu nazwę metody ustawiającej pole taką samą jak to pole, czy w tym przypadku byłoby to po prostu `id`

Reply:
poprawiono: 96619b9b46fa538703ca171c44c795010b4ffceb